src/org/labkey/test/tests/DatabaseDiagnosticsTest.java

```diff
@@ -71,10 +80,46 @@ public void validateDomainsTest()
                 .assertLogTextContains("Check complete, 0 errors found");
     }
 
+    @BeforeClass
+    public static void setupProject()
+    {
+        DatabaseDiagnosticsTest init = getCurrentTest();
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        _containerHelper.createProject(PROJECT_NAME, null);
+        _containerHelper.enableModules(Arrays.asList("Wiki"));
+
+        goToProjectHome();
+        PortalHelper portalHelper = new PortalHelper(this);
+        portalHelper.addBodyWebPart("Wiki");
```

Comment:
Adding the wiki webpart makes the setup longer and makes the test likely to trigger the CSP just by navigating through the project home page. Better to `goToModule("Wiki")` to create the wiki.